pkg/core/store_option.go

```diff
@@ -98,6 +98,15 @@ func SetStoreState(state metapb.StoreState, physicallyDestroyed ...bool) StoreCr
 	}
 }
 
+// SetNodeState sets the node state for the store.
+func SetNodeState(nodeState metapb.NodeState) StoreCreateOption {
```

Comment:
Can we directly use SetStoreState?

Reply:
<img width="975" height="605" alt="{6350338C-F518-49C0-89D1-2CF94117429E}" src="https://github.com/user-attachments/assets/3608684d-114f-4ac5-9ecd-d863ea900713" />